src/include/btmem.h

```diff
@@ -712,7 +700,17 @@ struct __wt_page {
                                           /* AUTOMATIC FLAG VALUE GENERATION STOP 16 */
     uint16_t flags_atomic;                /* Atomic flags, use F_*_ATOMIC_16 */
 
-    uint8_t unused; /* Unused padding */
+#define WT_PAGE_IS_INTERNAL(page) \
+    ((page)->type == WT_PAGE_COL_INT || (page)->type == WT_PAGE_ROW_INT)
+#define WT_PAGE_INVALID 0       /* Invalid page */
+#define WT_PAGE_BLOCK_MANAGER 1 /* Block-manager page */
+#define WT_PAGE_COL_FIX 2       /* Col-store fixed-len leaf */
+#define WT_PAGE_COL_INT 3       /* Col-store internal page */
+#define WT_PAGE_COL_VAR 4       /* Col-store var-length leaf page */
+#define WT_PAGE_OVFL 5          /* Overflow page */
+#define WT_PAGE_ROW_INT 6       /* Row-store internal page */
+#define WT_PAGE_ROW_LEAF 7      /* Row-store leaf page */
+    uint8_t type;               /* Page type */
 
```

Comment:
I would keep the `uint8_t unused;` as a reminder to future readers that there is another byte of padding in here. It also matches the way we annotate such padding in other places.

Reply:
Thank you both for the approval. Would a comment suffice instead of an explicit padding field?
Note: I'm just waiting for OOB discussions to conclude before I make the change.

cc @sueloverso

Reply:
I'm good with whatever the majority decided on slack. Both have pros/cons and work for me.

Reply:
I also don't have strong feelings on this point. But I do think it's good to be consistent across the code. I would suggest including the explicit `unused` field in this PR for consistency.

Then you can make the case for comments on compiler padding, or some other alternative, and create a new ticket/PR to convert to the resulting standard.